extensionsGallery.json

```diff
@@ -3358,7 +3358,64 @@
 					],
 					"statistics": [],
 					"flags": ""
-				}
+				},
+				{
+				"extensionId": "72",
+				"extensionName": "eltsnap-simple-data-flow",
+				"displayName": "eltsnap: Simple Data Flow",
+				"shortDescription": "Easily copy a table from one SQL Server to Another",
+				"publisher": {
+				  "displayName":"Jim-BITracks",
+				  "publisherId": "bitracks",
+				  "publisherName":"bitracks"
+				},
+				"versions": [
+				  {
+					"version": "1.0.0",
+					"lastUpdated": "8/31/2020",
+					"assetUri": "",
+					"fallbackAssetUri": "fallbackAssetUri",
+					"files": [
+						{
+							"assetType": "Microsoft.VisualStudio.Services.VSIXPackage",
+							"source": "https://github.com/Jim-BITracks/eltsnap-simpleDataFlow/releases/tag/v1.0.0"
+						},
+						{
+							"assetType": "Microsoft.VisualStudio.Services.Icons.Default",
+							"source": "https://raw.githubusercontent.com/Jim-BITracks/eltsnap-simpleDataFlow/v1.0.0/images/bitracks.png"
+						},
+						{
+							"assetType": "Microsoft.VisualStudio.Services.Content.Details",
+							"source": "https://raw.githubusercontent.com/Jim-BITracks/eltsnap-simpleDataFlow/v1.0.0/README.md"
+						},
+						{
+							"assetType": "Microsoft.VisualStudio.Code.Manifest",
+							"source": "https://raw.githubusercontent.com/Jim-BITracks/eltsnap-simpleDataFlow/v1.0.0/package.json"
+						},
+						{
+							"assetType": "Microsoft.VisualStudio.Services.Content.License",
+							"source": "https://raw.githubusercontent.com/Microsoft/azuredatastudio/main/LICENSE.txt"
+						}
+					],
+					"properties": [
+						{
+							"key": "Microsoft.VisualStudio.Code.ExtensionDependencies",
+							"value": ""
+						},
+						{
+							"key": "Microsoft.VisualStudio.Code.Engine",
```

Comment:
Please also add an AzDataEngine section. If you aren't sure what versions the extension supports you should probably leave it as >=1.21.0 unless you've specifically tested with earlier versions. 

```
{
	"key": "Microsoft.AzDataEngine",
	"value": ">=1.21.0"
},
```